MMM-plex-recently-added.js

```diff
@@ -102,8 +102,13 @@ Module.register('MMM-plex-recently-added', {
 
   _getThumbUrl(item) {
     var thumbKey = 'thumb';
-    if (item.type === 'episode') {
+    if (item.type === 'episode' | item.type === 'season') {
```

Comment:
Should probably be `||`? Otherwise it's a bitwise OR

Reply:
Yes. You are correct.